Intersect.Client.Framework/Plugins/Interfaces/IClientLifecycleHelper.cs

```diff
@@ -25,9 +26,33 @@ public class GameUpdateArgs : EventArgs
     {
         public GameStates State { get; }
 
-        public GameUpdateArgs(GameStates state)
+        public IEntity Player { get; }
+
+        public Dictionary<Guid, IEntity> KnownEntities { get; }
```

Comment:
Is a dictionary needed? If it is, how about a readonly copy (and use the IReadOnlyDictionary interface instead of Dictionary) instead of the actual dictionary?

Reply:
Easier to reference things by Id rather than having to loop through it to find the right Entity if you're tracking one between updates.

Can definitely make it read only though